specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -136,12 +134,12 @@ For example, since EJB Lite does not include
 any remote functionality, the _EJB_ annotation may not be used to inject
 a remote reference, something that should be kept in mind when
 evaluating the requirements in the Platform specification section
-“Enterprise JavaBeans References”.
+“Enterprise JakartaBeans References”.
 
 === Acknowledgements for Version 6
 
 Version 6 of this specification was created
-under the Java Community Process as JSR-316. The spec leads for the
+under the Jakarta EE Specification Process as JSR-316. The spec leads for the
```

Comment:
This is about Java EE 6, which is not Jakarta. Should be kept untouched. Same for version 7 and 8 of Java EE.

Reply:
It would probably be best to remove all the acknowledgements for versions of the Java EE specification and add acknowledgements for the Jakarta EE 8 specification, including how
it is derived from the Java EE 8 specification contributed by Oracle to the Eclipse Foundation.